sdk/internal/perf/perf.go

```diff
@@ -12,6 +12,25 @@ import (
 	"runtime"
 )
 
+func init() {
+	// Start with adding all of our arguments
+	flag.IntVar(&duration, "d", 10, "Duration of test in seconds.")
```

Comment:
If you threw away the short options that would actually be allright by me, mostly because now the options are self-documenting in reviews.

(no big deal either way)

Reply:
Keeping them for now because this is how other languages do them, we might need them for language agnostic automation, if not I can remove them later.